Mapbox/MapboxMapsStyle/Types/Color.swift

```diff
@@ -1,48 +1,34 @@
 import UIKit
 
-public typealias ColorRepresentable = String
-extension ColorRepresentable {
+/// Container to represent `UIColor`for use by the map renderer
+public struct ColorRepresentable: Codable, Equatable {
+
+    /// String representation of a `UIColor` used by the renderer
+    public let colorRepresentation: String?
 
     /// Create a string representation of a `UIColor`
-    public init(color: UIColor) {
+    /// - Parameter color: A `UIColor` instance in the sRGB color space
+    /// - Returns: Initializes a `ColorRepresentable` instance if the `color` is in sRGB color space. Returns `nil` otherwise.
```

Comment:
Technically, the color only needs to be in a colorspace that is "compatible" with the "extended sRGB" color space for the `getRed(_:green:blue:alpha:)` call to succeed.

https://developer.apple.com/documentation/uikit/uicolor/1621919-getred

Reply:
Yep, but the style spec only supports `sRGB` color space -- https://docs.mapbox.com/mapbox-gl-js/style-spec/types/#color